snuba/datasets/storages/transactions.py

```diff
@@ -133,6 +133,24 @@ def transactions_migrations(
             f"AFTER measurements.key"
         )
 
+    if "url" not in current_schema:
```

Comment:
Same question I had on the error table.
The url column inn the events table is a promoted tag, that means the column exists onnly for optimization and should already be accessible as `tags[url]`. 
If we are preserving that behavior there should be no need to add the column here and we could rely on the tag.

https://github.com/getsentry/snuba/blob/master/snuba/datasets/storages/events_common.py#L41-L54